shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-postgresql/src/main/java/org/apache/shardingsphere/data/pipeline/postgresql/prepare/datasource/PostgreSQLDataSourcePreparer.java

```diff
@@ -132,12 +137,12 @@ private String rewriteActualIndexSql(final String sql, final String actualTableN
                 try (Connection sourceConnection = dataSource.getConnection()) {
                     String actualTableName = dataNode.getTableName();
                     StringJoiner joiner = new StringJoiner(";");
-                    Pair<String, List<String>> pkPiar = queryTablePrimaryKey(sourceConnection, actualTableName);
-                    joiner.add(queryCreateTableSql(sourceConnection, actualTableName, pkPiar.getRight()));
-                    queryCreateIndexes(sourceConnection, actualTableName, pkPiar.getLeft()).forEach(joiner::add);
-                    String logicTableName = each.getLogicTableName();
+                    Pair<String, List<String>> pkPair = queryTablePrimaryKey(sourceConnection, actualTableName);
+                    joiner.add(queryCreateTableSql(sourceConnection, actualTableName, pkPair.getRight()));
+                    queryCreateIndexes(sourceConnection, actualTableName, pkPair.getLeft()).forEach(joiner::add);
```

Comment:
It's better to use `primaryKey` to replace `pk`.

Reply:
done , changed